lib/webpack.config.js

```diff
@@ -46,6 +46,26 @@ const babelLoader = (plugins = []) => ({
   },
 })
 
+const packageWhitelist = ['catalog-ui-search', 'imperio', '@connexta']
+
+const includeInWebpackBuild = modulePath => {
+  // Include files not in node_modules
+  if (!modulePath.includes('node_modules')) {
+    return true
+  }
+
+  // Include files in whitelisted packages
+  return packageWhitelist.some(pkg => {
+    return (
+      modulePath.includes(pkg) &&
+      // Don't include items in nested node_modules
+      !modulePath.includes(pkg + '/node_modules')
```

Comment:
Just ran into an issue with this.
It resolves to `@connexta/node_modules` which is never the case.
I swapped it to `modulePath.lastIndexOf(pkg) > modulePath.lastIndexOf('node_modules')` and it started working again.